aiida/cmdline/commands/cmd_setup.py

```diff
@@ -18,8 +20,11 @@
 from aiida.manage.configuration import load_profile
 from aiida.manage.manager import get_manager
 
+CLI_LOGGER = logging.getLogger('cli')
```

Comment:
should we maybe define this once in `aiida.common.log` such that it can just be imported. Should we want or need to change the logger name at some point, that would make it easier. We also do this for `AIIDA_LOGGER`.

Reply:
Sure, we can do this.